inc/class-emitter.php

```diff
@@ -41,6 +41,13 @@ class Emitter {
 	const HEADER_KEY = 'Surrogate-Key';
 
 	/**
+	 * Maximum header length.
+	 *
+	 * @var integer
+	 */
+	const HEADER_MAX_LENGTH = 4096;
```

Comment:
It was easy to test. I've upped that limit in the next PR update (and fixed a typo that caused the unit tests to fail).